subscription-engine/kafka-ws/app/App.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+
+const expressWS = require('express-ws');
+
+const addRequestId = require('express-request-id');
+
+const compression = require('compression');
+
+const createError = require('http-errors');
+
+const morgan = require('morgan');
+
+/* This library is about what happens when you hit an async error. */
+require('express-async-errors');
```

Comment:
Should this be only declared, with no variable assignment?

Reply:
Yes sir!
https://www.npmjs.com/package/express-async-errors